ccm_web/server/src/migrations/2021.06.02T15.59.37.create-canvas-token.ts

```diff
@@ -0,0 +1,30 @@
+import { DataTypes } from 'sequelize'
+import { Migration } from '../migrator'
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().createTable('canvas_token', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.BIGINT
+    },
+    userId: {
+      allowNull: false,
+      type: DataTypes.BIGINT,
+      references: { model: 'user', key: 'id' }
```

Comment:
why can't we reference `login_id` instead of this auto-generated user `id`?

Reply:
I believe it's a best practice to join tables on a PK?

Reply:
I just felt since `loginid` has unique constraint instead of random id's we can directly use uniqnames in canvasToken table so that way it is recognizable who the token belongs to. But You could have multiple Unique constraint in a colunm so going by Id for join is fine perhaps.

Reply:
You could do it the way you are proposing, but I think I prefer it this way. Determining who the token belongs to is as simple as joining the tables, so I don't think we particularly need the login ID to also be in the `canvas_token` table.